core/src/main/java/tc/oc/pgm/variables/BlitzVariable.java

```diff
@@ -0,0 +1,38 @@
+package tc.oc.pgm.variables;
+
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.blitz.BlitzMatchModule;
+import tc.oc.pgm.filters.Filterable;
+
+public class BlitzVariable implements Variable<MatchPlayer> {
+
+  private final VariableDefinition<MatchPlayer> definition;
+
+  public BlitzVariable(VariableDefinition<MatchPlayer> definition) {
+    this.definition = definition;
+  }
+
+  @Override
+  public VariableDefinition<MatchPlayer> getDefinition() {
+    return definition;
+  }
+
+  @Override
+  public double getValue(Filterable<?> context) {
+    MatchPlayer matchPlayer = (MatchPlayer) context;
+    BlitzMatchModule blitzMatchModule = matchPlayer.getMatch().needModule(BlitzMatchModule.class);
+    return blitzMatchModule.getNumOfLives(matchPlayer.getId());
+  }
+
+  @Override
+  public void setValue(Filterable<?> context, double value) {
+    MatchPlayer matchPlayer = (MatchPlayer) context;
```

Comment:
```suggestion
    MatchPlayer matchPlayer = context.getFilterableAncestor(MatchPlayer.class);
```
although we don't have a children that could have match player as parent and this will always return just itself, it's a safer method (won't get an illegal cast).
(note: same for getValue)